config/samples/v2/dspa_all_fields.yaml

```diff
@@ -0,0 +1,214 @@
+# This file should not be used to deploy a DataSciencePipelinesApplication
+# It's main purpose is to show all possible fields that can be configured
+# Note that you cannot specify all fields, some are mutually exclusive
+# For example, you can only specify either a miniodb deployment or
+# externalstorage connection, but not both
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: sample
+  namespace: data-science-project
+spec:
```

Comment:
can you remove the fields pointed above as not being relevant for v2 here as well

Reply:
`metadata` and `spec`? Are you sure?

Reply:
ignore, I think I highlighted the wrong portion